frontends/mit-open/src/page-components/Header/UserMenu.tsx

```diff
@@ -83,25 +125,64 @@ const UserMenu: React.FC = () => {
     },
   ]
 
-  return (
-    <SimpleMenu
-      onVisibilityChange={setVisible}
-      items={items
-        .filter(({ allow }) => allow)
-        .map(({ allow, ...item }) => item)}
-      trigger={
-        <StyledBadge
-          overlap="circular"
-          anchorOrigin={badgeAnchorOrigin}
-          badgeContent={visible ? <ArrowDropUpIcon /> : <ArrowDropDownIcon />}
-        >
-          <SmallAvatar component="button" aria-label="User Menu">
-            {user?.is_authenticated ? <UserIcon user={user} /> : <PersonIcon />}
-          </SmallAvatar>
-        </StyledBadge>
-      }
-    ></SimpleMenu>
-  )
+  const menuOverrideProps: MenuOverrideProps = {
+    anchorOrigin: { horizontal: "right", vertical: "bottom" },
+    transformOrigin: { horizontal: "right", vertical: "top" },
+  }
+
+  if (user?.is_authenticated) {
+    return (
+      <SimpleMenu
+        menuOverrideProps={menuOverrideProps}
+        onVisibilityChange={setVisible}
+        items={items
+          .filter(({ allow }) => allow)
+          .map(({ allow, ...item }) => item)}
+        trigger={
+          <UserMenuContainer role="button" aria-label="User Menu">
+            <UserIcon data-testid="UserIcon" />
+            <UserName user={user} />
+            {user?.is_authenticated ? <UserMenuChevron open={visible} /> : ""}
+          </UserMenuContainer>
+        }
+      />
+    )
+  } else {
+    return (
+      <LoginButtonContainer data-testid="login-button-container">
+        {variant === "desktop" ? (
+          <FlexContainer className="login-button-desktop">
+            <ButtonLink
```

Comment:
The design seems to have been updated since you coded this up to have separate buttons for sign up and login :neutral_face:
<img width="376" alt="image" src="https://github.com/mitodl/mit-open/assets/939376/c6a38c71-061b-4ab9-a65b-07ea2924db86">

Side note for designers - "Log In" please if it's a call to action, not "Login" (we have "Sign Up" next door). @steven-hatch